AmplifyPlugins/Auth/Sources/AWSCognitoAuthPlugin/Task/Protocols/AmplifyAuthTask.swift

```diff
@@ -31,11 +31,11 @@ extension AmplifyAuthTask where Self: DefaultLogger {
             do {
                 log.info("Starting execution for \(eventName)")
                 let valueReturned = try await execute()
-                log.info("Successfully completed execution for \(eventName) with result:\n\(prettify(valueReturned))")
```

Comment:
where is `prettyify` coming from now?

Reply:
oh nvm, just noticed we're removing the usage of it

Reply:
synced offline, `prettyify`'s output was not redacted, removing usage of it now will redact it properly